monocle/static/js/main.js

```diff
@@ -104,6 +105,7 @@ function getPopupContent (item) {
         content += '<a href="#" data-pokeid="'+item.pokemon_id+'" data-newlayer="Pokemon" class="popup_filter_link">Move to Pokemon</a>';
     }else{
         content += '<a href="#" data-pokeid="'+item.pokemon_id+'" data-newlayer="Trash" class="popup_filter_link">Move to Trash</a>';
+	content += ' /<a href=https://www.google.com/maps/?daddr='+ item.lat + ','+ item.lon +' target="_blank" title="See in Google Maps">Get directions</p></a>';
```

Comment:
1. tab / spaces
2. this way it will only be shown if it is in the Pokemon group, I think it would be better to have it for Trash too
3. above "|" is used as spacer, you are using a "/"